lib/src/utils.dart

```diff
@@ -30,6 +30,10 @@ Map/*<K, V>*/ updateMap/*<K, V>*/(
 ///     mapToQuery({"foo": "bar", "baz": "bang"});
 ///     //=> "foo=bar&baz=bang"
 String mapToQuery(Map<String, String> map, {Encoding encoding}) {
+  map.keys
```

Comment:
Both done. The tests were super simple changes, so I don't expect any problems, but, I don't have sdk v1.23 installed due to local requirements for 1.22, so I couldn't *actually* run the test suite.